src/common/store/session/actions.ts

```diff
@@ -1,4 +1,4 @@
-import Web3 from 'web3';
+// import Web3 from 'web3';
```

Comment:
I'll would remove this line if is no longer needed

Reply:
removed